### ISSUE TITLE
Bad request for A4

### ISSUE THREAD

**Issue by Kailai-Wang** [litentry/litentry-parachain#1836]
### Context

See the [slack thread](https://litentry.slack.com/archives/C05328L0GAF/p1687878930338309)
Is it a problem of achainable or our request construction?

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

**Comment by outofxxx** [comment]
Here's the response message from TDF `the holder request`.

```json
{
    "errors": [
        {
            "message": "Value \"litentryrococo\" does not exist in \"VerifiedCredentialsNetwork\" enum. Did you mean the enum value \"litentry\"?",
            "locations": [
                {
                    "line": 1,
                    "column": 143
                }
            ]
        }
    ]
}
```
`litentryrococo` network is not supported.

**Comment by Kailai-Wang** [comment]
Then shall we exclude `LitentryRococo` from supported networks?

**Comment by zTgx** [comment]
Yes,I think so. Better to exclude it.

**Comment by Kailai-Wang** [comment]
I think this will be resolved once #1854 is merged:
```
// LIT holder, not including `LitentryRococo` as it's not supported by any data provider
Self::A4(..) => vec![Web3Network::Litentry, Web3Network::Litmus, Web3Network::Ethereum],
```

but let's re-test it later

**Comment by Kailai-Wang** [comment]
Should be fixed in #1854
